packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/tile_overlay.dart

```diff
@@ -0,0 +1,159 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'types.dart';
+import 'package:meta/meta.dart' show immutable, required;
+
+/// Uniquely identifies a [TileOverlay] among [GoogleMap] tile overlays.
+///
+/// This does not have to be globally unique, only unique among the list.
```

Comment:
Among which list?

Reply:
I think this is a copy-paste from other objects in google maps. This is confusing, I removed it.